addons/mail/static/src/discuss/discuss.xml

```diff
@@ -9,9 +9,19 @@
             <!-- top bar -->
             <div class="px-3 bg-100 border-bottom d-flex flex-shrink-0" style="height:42px;">
                 <ThreadIcon className="'me-2 align-self-center'" thread="thread"/>
-                <div class="d-flex flex-grow-1 align-self-center align-items-center">
-                    <div class="lead flex-shrink-0 px-1 border text-truncate fw-bold border-light" t-att-title="thread.name" >
-                        <t t-esc="thread.name"/>
+                <div class="d-flex flex-grow-1 align-self-center align-items-center h-100 py-2">
+                    <div class="o-mail-discuss-thread-name-container">
+                        <span class="lead fw-bold px-2" t-out="state.threadRename"/>
```

Comment:
Would be best to define an `AutogrowInput` component.
So that it's reusable, and this trick of invisible `<span>` is not visibly in DOM of component other than this `AutogrowInput`.